torchci/components/FilteredJobList.tsx

```diff
@@ -7,6 +7,37 @@ import JobLinks from "./JobLinks";
 import JobSummary from "./JobSummary";
 import LogViewer from "./LogViewer";
 import { JobAnnotation } from "lib/types";
+import useScrollTo from "lib/useScrollTo";
```

Comment:
You found it!! This is the function Kerry told me to do the scrolling after the page is loaded :D